commands/issue/update/issue_update.go

```diff
@@ -96,6 +121,51 @@ func NewCmdUpdate(f *cmdutils.Factory) *cobra.Command {
 					l.MilestoneID = gitlab.Int(0)
 				}
 			}
+			if len(assigneesToReplace) != 0 {
+				users, err := api.UsersByNames(apiClient, assigneesToReplace)
```

Comment:
I was going to comment about this part but it's been fixed. That was what caused `4`. 
`assigneesToAdd` was used instead

Reply:
I think that was what was causing `3`. the `4` perplexes me because There is no reason it should fail to add oneself again, you are just sending a `[]int{profclems.ID, profclems.ID}`. GitLab people surely worked around that and automatically remove any duplicates.

Reply:
Sorry I meant `3`. I can still reproduce `4`

Reply:
Seems that's an issue from GitLab

Reply:
Magically, it works now. Issue `4` is resolved